Thesis.tex

```diff
@@ -64,9 +64,47 @@ \section{Patents}
 \section{Trademark}
 \label{sec:Trademark}
 
-\section{Organisations}
+\section{Notable Organisations}
 \label{sec:Orgas}
 
+\subsection{BSA}
+\label{ssec:BSA}
+The Software Alliance (changed its name from \underline{B}usiness \underline{S}oftware \underline{A}lliance) is an advocate group representing the interests of the software industry in countries around the world. It was established in 1988 and has its headquarters in Washington DC. Notable members include, amongst others, Adobe, Apple, Autodesk, IBM, Microsoft, Oracle, and Siemens. The current CEO, Victoria Espinel, studied to become a telecommunications lawyer before being appointed as advisor on intellectual property issues of President Obama in 2009 and subsequent appointment to her current position inside the BSA. \parencite{Rogers2017}
```

Comment:
before being appointed [] and subsequent appointment ... doesn't add up
... and subsequently being appointed ...
oder
... and her subsequent appointment ...